docs/services/identity.md

```diff
@@ -47,48 +50,89 @@ The identity service is locate under [`token/services/identity`](./../../token/s
 
 Building on the concept of long-term identities, we'll now explore how they are grouped into roles within the identity service.
 
-Each role acts as a container for long-term identities, which are then used to create wallets. Here's the interface that defines a role:
+Each role acts as a container for long-term identities, which are then used to create wallets. 
+Here's the interface that defines a role:
 
 ```go
 // Role is a container of long-term identities.
 // A long-term identity is then used to construct a wallet.
 type Role interface {
```

Comment:
Maybe we could call it `IdentityManager`, `IdentityPool`, `IdentityContainer` with a method `func Role() RoleType`. `Role` is a bit confusing